packages/tenderly-core/src/internal/core/common/logger.ts

```diff
@@ -12,12 +15,34 @@ export function logApiError(err: any) {
       message = err.response.data;
     }
 
-    console.log(API_ERR_MSG, code, codeText, message);
+    logger.error(API_ERR_MSG, code, codeText, message);
     return;
   }
 
   // general error
   if (err instanceof Error) {
-    console.log(err.message);
+    logger.error(err.message);
   }
 }
+
+export function logConfig(config: TenderlyConfig) {
+  logger.trace("Value of the config:", {
```

Comment:
do we need to log the entire config with all potential sensitive data/values?
Anyway, just make sure access tokens and other sensitive data are not logged.